### DISCUSSION TITLE
if we compare ntopng with this project then what do you feel about this project

### DISCUSSION THREAD

**Discussion by Greatz08** [GyulyVGC/sniffnet#477] [General]
Is it better or same or less capable than ntopng?

**Comment by GyulyVGC** [comment]
Hi @HakaishinShwet, thanks for your interest!

It really depends on your use case.

Do you need something simple to keep an eye on your traffic? Go for Sniffnet.
Do you need something with more advanced features to deeply inspect your traffic? Go for ntopng.

In terms of number of functionalities ntopng is much more rich, being it developed by a team of devs for several years now.

Sniffnet is being actively maintained and new features are constantly being added, although its main focus will remain on the ease of use: it includes the key features necessary to keep track of your traffic without introducing the complexity that characterises Wireshark and other network monitoring tools.

**Reply by Greatz08** [reply]
@GyulyVGC thanks for explaining in simple way,i also was expecting similar response.
Anyways keep up the good work, Thanks for this foss solution :-))
